cogeo_mosaic/backends/base.py

```diff
@@ -1,14 +1,17 @@
 """cogeo_mosaic.backend.base: base Backend class."""
 
+from typing import Dict, Tuple
+
```

Comment:
I think the default `isort` behavior is group imports into 

1. Standard lib
2. External packages
3. Internal modulels

Within each group, put all `import`s before all `from ... import`

And then alphabetize within each subgroup.

Reply:
yeah I will tend to agree with isort, but let's keep the ordering as similar as is the other module for now.